server/server.js

```diff
@@ -3527,6 +3527,68 @@ const sockets = (() => {
                               util.warn(`Ran ${args[1]} due to /eval being used in game.`)
                               eval(args[1])
                             break;
+                            case 'summon':
+                            case 'spawn':
```

Comment:
Brought back `/spawn` and `clone`, and added `/coords`, `/settank` and `/reset`.